log/development.log

```diff
@@ -80,3 +80,58 @@ FOREIGN KEY ("post_id")
   [1m[36mTRANSACTION (0.8ms)[0m  [1m[35mCOMMIT[0m
   [1m[35m (0.3ms)[0m  [1m[34mSELECT pg_advisory_unlock(1365696269427442995)[0m
   [1m[36mActiveRecord::SchemaMigration Pluck (0.6ms)[0m  [1m[34mSELECT "schema_migrations"."version" FROM "schema_migrations" ORDER BY "schema_migrations"."version" ASC[0m
+Started GET "/" for ::1 at 2022-12-05 18:41:39 +0530
+  [1m[36mActiveRecord::SchemaMigration Pluck (5.8ms)[0m  [1m[34mSELECT "schema_migrations"."version" FROM "schema_migrations" ORDER BY "schema_migrations"."version" ASC[0m
+Processing by Rails::WelcomeController#index as HTML
```

Comment:
[OPTIONAL] Kindly connect your MIT license to your description correctly. Because, If I try to open it, it shows me a broken link.

<img width="800" alt="Screenshot 2022-12-05 at 18 41 06" src="https://user-images.githubusercontent.com/69011963/205652095-f214e72e-2d6e-4203-81a6-fe209719384e.png">